src/Symfony/Component/Cache/Tests/Adapter/CouchbaseBucketAdapterTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Tests\Adapter;
+
+use Symfony\Component\Cache\Adapter\AbstractAdapter;
+use Symfony\Component\Cache\Adapter\CouchbaseBucketAdapter;
+use Symfony\Component\Cache\Exception\InvalidArgumentException;
+
+/**
+ * @author Antonio Jose Cerezo Aranda <aj.cerezo@gmail.com>
+ */
+class CouchbaseBucketAdapterTest extends AdapterTestCase
+{
+    protected $skippedTests = [
+        'testClearPrefix' => 'Couchbase cannot clear by prefix',
+    ];
+
+    /** @var \CouchbaseBucket */
+    protected static $client;
+
+    /**
+     * @requires extension couchbase 2.6.0
+     */
+    public static function setupBeforeClass(): void
+    {
+        self::$client = AbstractAdapter::createConnection('couchbase://'.getenv('COUCHBASE_HOST').'/cache',
+            ['username' => getenv('COUCHBASE_USER'), 'password' => getenv('COUCHBASE_PASS')]
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function createCachePool($defaultLifetime = 0)
+    {
+        $client = $defaultLifetime
+            ? AbstractAdapter::createConnection('couchbase://'
+                .getenv('COUCHBASE_USER')
+                .':'.getenv('COUCHBASE_PASS')
+                .'@'.getenv('COUCHBASE_HOST')
+                .'/cache')
+            : self::$client;
+
+        return new CouchbaseBucketAdapter($client, str_replace('\\', '.', __CLASS__), $defaultLifetime);
+    }
+
+    /**
+     * @test
+     */
+    public function createConnectionShouldThrowInvalidArgumentException(): void
```

Comment:
```suggestion
    public function testCreateConnectionShouldThrowInvalidArgumentException(): void
```

and remove the `@test`annotation

Reply:
This is done.